sdk/go/hcp/pulumiUtilities.go

```diff
@@ -83,3 +83,17 @@ func isZero(v interface{}) bool {
 	}
 	return reflect.ValueOf(v).IsZero()
 }
+
```

Comment:
@stack72 After playing around with some of this code, the commands in the `Makefile` will regenerate all the SDK files, except for this file. The comment at the top of the file implies that it should be generated by the `tfgen` tool, though.

How is this file actually managed?